### DISCUSSION TITLE
New ACG logo

### DISCUSSION THREAD

**Discussion by ftessier** [a-cosmology-group/acg#105] [General]
While working on a higher resolution logo for acg, I came across this idea for a _new_ logo. This is a rough mockup, showing the idea. It could be stylized further. It evokes radio waves and (with some imagination) a galaxy. What do you think?

![acg](https://github.com/a-cosmology-group/acg/assets/1444241/0dd352eb-8f6c-498d-9882-52ec056645a7)

Using a [single storey "a"](https://en.wikipedia.org/wiki/A) would perhaps reinforce the circular symmetry:

![acg3](https://github.com/a-cosmology-group/acg/assets/1444241/00d77bfd-57e8-4901-9c1c-b47e25135e27)

**Comment by sahil5d** [comment]
I like it! Can the letters have constant width between them? What font would achieve that...

https://fonts.google.com/?preview.text=a%20C%20G&preview.text_type=custom&category=Sans+Serif,Display,Monospace

**Reply by ftessier** [reply]
Oh, this is achieved in the drawing software, practically no font will do that out of the box. For the mock up I used the Mulish google font, since it had a single-story `a` and a simple geometric shape. The logo can then be tweaked at will as a vector drawing. (The first logo used the google font Raleway).

**Comment by ftessier** [comment]
Here is a stylized version, but it lacks dynamics, it is too symmetric:

![acg4](https://github.com/a-cosmology-group/acg/assets/1444241/dd432166-57e8-46ee-8861-824a9d98d049)

**Comment by ftessier** [comment]
Here are two stylized versions that are more dynamic:

![acg5](https://github.com/a-cosmology-group/acg/assets/1444241/24af068f-f7a4-41e9-8c66-e69b6157188c) 
![acg6](https://github.com/a-cosmology-group/acg/assets/1444241/68e51794-815d-4ad4-80c3-bab87d82508f)

The first one evokes radio waves and the Doppler shift!

The second one is more dynamic visually (even a bit disturbing, which is perfect for this group 😉). It is also more "readable" (owing to the larger `G` thickness on the right). It is also very balanced graphically speaking (largest circle is concentric with inner one).

Both are built using primitive shapes only, so it is easy to reproduce and save as a very small `.svg` graphic file.

**Reply by RedshiftDrift** [reply]
Once it is reduced in size it may just appear like a blob/spot that is too dense.  Perhaps use thinner 'C' and 'G'?
Add some colours?  blue/purple-'a', green='C', and red-'G' ?
(That would make it look like a spectrum - personal bias, like spectroscopic redshift 🤣 )

**Reply by ftessier** [reply]
I think it is better to stick with one colour for a logo, it is then easy to change the colour scheme 😆. Will check for reduced size and adjust accordingly. Do you like it otherwise?

**Reply by RedshiftDrift** [reply]
I like this version 
![ACG](https://github.com/a-cosmology-group/acg/assets/16748402/7bc7d971-0297-4bcf-8b7f-85d6c0494acb)
as long as the letter-thickness is reduced to well separate the letters on the small logo.

**Reply by ftessier** [reply]
I see. Ok, will work towards a final version that is clearer at 32x32 and a few minor improvements for the larger version. Since the reference version will be svg (vector), it will be scalable at will.

**Comment by ftessier** [comment]
Here is my "final" 😄 version:

![acg](https://github.com/a-cosmology-group/acg/assets/1444241/3ddf73ca-21db-4bf1-86b7-1998afe83be1)

Looks ok in 32x32:

![tmp](https://github.com/a-cosmology-group/acg/assets/1444241/47350dfa-107c-43ae-b47e-813ec6777cac)

I will add the `.svg` and the `.png` in the `assets/` directory. Here is the svg code for the record:

```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
	xmlns="http://www.w3.org/2000/svg"
	xmlns:xlink="http://www.w3.org/1999/xlink" style="isolation:isolate" viewBox="0 0 512 464" width="512pt" height="464pt">
	<defs>
		<clipPath id="_clipPath_8brULbIyX8qbl3zq5bbZTpzBIH0kNkkN">
			<rect width="512" height="464"/>
		</clipPath>
	</defs>
	<g clip-path="url(#_clipPath_8brULbIyX8qbl3zq5bbZTpzBIH0kNkkN)">
		<g>
			<path d=" M 364.635 264.088 L 365.394 269.469 C 365.682 271.51 367.753 272.909 370.012 272.593 L 394.585 269.153 C 396.846 268.837 398.447 266.923 398.159 264.882 L 383.553 161.358 C 383.265 159.317 381.197 157.918 378.936 158.234 L 354.363 161.675 C 352.104 161.991 350.501 163.904 350.789 165.945 L 351.548 171.327 C 339.245 164.894 324.441 162.093 309.087 164.243 C 272.921 169.306 247.298 199.925 251.904 232.573 L 251.904 232.573 C 256.51 265.222 289.613 287.619 325.779 282.556 C 341.133 280.407 354.589 273.65 364.635 264.088 Z  M 358.654 221.697 L 357.529 213.718 C 353.333 199.578 338.036 190.352 321.451 192.674 C 303.367 195.206 290.556 210.514 292.86 226.84 L 292.86 226.84 C 295.163 243.165 311.713 254.363 329.797 251.831 C 346.382 249.509 358.534 236.44 358.654 221.697 Z " fill-rule="evenodd" fill="rgb(128,0,161)"/>
			<path d=" M 400.117 151.499 C 400.911 151.387 416.61 149.19 419.171 148.831 C 422.273 148.389 423.646 145.053 422.021 142.965 C 389.747 101.472 333.175 77.923 273.038 86.343 C 187.141 98.368 126.288 171.085 137.228 248.628 L 137.228 248.628 C 148.169 326.171 226.785 379.362 312.682 367.337 C 372.827 358.916 420.695 320.739 440.234 271.983 C 441.347 268.918 438.282 266.806 435.847 267.147 L 416.889 269.801 C 415.546 269.989 414.53 270.665 413.823 271.723 C 396.967 298.383 367.3 318.217 331.517 323.226 C 270.487 331.77 214.626 293.976 206.853 238.881 L 206.853 238.881 C 199.08 183.785 242.319 132.117 303.349 123.573 C 339.132 118.563 373.138 129.483 396.715 150.48 C 397.668 151.253 399.322 151.61 400.117 151.499 Z " fill="rgb(128,0,161)"/>
			<path d=" M 499.136 164.045 C 500.577 163.842 502.719 164.52 503.501 166.641 C 506.325 176.295 508.484 186.244 509.924 196.451 C 511.519 207.752 512.179 218.989 511.959 230.099 C 509.712 343.372 415.969 443.364 288.341 461.232 C 148.197 480.852 19.924 394.065 2.075 267.549 L 2.075 267.549 C -15.774 141.034 83.515 22.388 223.658 2.768 C 339.227 -13.411 446.719 42.767 490.877 133.915 C 491.946 135.954 490.568 138.815 487.637 139.246 C 464.479 142.488 452.519 144.162 451.76 144.269 C 451 144.375 448.591 144.13 447.544 142.408 C 411.241 82.76 336.513 47.294 256.578 58.484 C 152.6 73.041 78.934 161.068 92.177 254.935 L 92.177 254.935 C 105.42 348.802 200.59 413.192 304.568 398.635 C 394.156 386.093 461.244 319.007 469.624 240.639 C 469.843 237.865 467.328 236.347 465.024 236.67 L 432.124 241.276 C 429.253 241.678 427.183 239.615 427.449 237.091 C 429.159 227.631 429.412 217.831 428.013 207.918 C 426.615 198.011 423.663 188.663 419.405 180.043 C 418.529 178.215 419.506 175.193 422.688 174.748 Q 422.703 174.738 422.718 174.727 L 460.664 169.414 L 460.664 169.414 L 499.136 164.045 Z " fill="rgb(128,0,161)"/>
		</g>
	</g>
</svg>
```

**Reply by RedshiftDrift** [reply]
... and with a slight angle up! 👏

**Comment by ftessier** [comment]
Yes, with some imagination you can almost see a telescope pointed at the heavens!

**Reply by RedshiftDrift** [reply]
Remember that show? 😆 
![top](https://github.com/a-cosmology-group/acg/assets/16748402/764e6fbc-3b76-41e5-960b-fc129246a3d5)

**Reply by ftessier** [reply]
No, too young... 😉 😆

**Comment by ftessier** [comment]
Please don't merge just yet, I am uploading cleaner files...

**Comment by ftessier** [comment]
Pull request #106 is ready.
